dev/io.openliberty.org.apache.myfaces.4.0_fat/fat/src/io/openliberty/org/apache/myfaces40/fat/tests/SelectItemTests.java

```diff
@@ -0,0 +1,117 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.org.apache.myfaces40.fat.tests;
+
+import static org.junit.Assert.assertTrue;
+
+import java.net.URL;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+/**
+ *  Verifies that the f:selectItemGroup and f:selectItemGroup output the correct selections.
+ */
+@RunWith(FATRunner.class)
+public class SelectItemTests {
+
+    protected static final Class<?> clazz = SelectItemTests.class;
+
+    private static final String APP_NAME = "SelectItemTests";
+
+    @Server("faces40_selectItemsServer")
+    public static LibertyServer server;
+
+    @Rule
+    public TestName name = new TestName();
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        ShrinkHelper.defaultDropinApp(server, APP_NAME + ".war", 
+            "io.openliberty.org.apache.faces40.fat.beans.selectitemgroup",
```

Comment:
How would you feel about changing the order of the package a bit `io.openliberty.org.apache.faces40.fat.selectitemgroup.beans`